src/main/java/com/nylas/NeuralOcr.java

```diff
@@ -0,0 +1,32 @@
+package com.nylas;
+
+import java.util.List;
+
+public class NeuralOcr extends File {
```

Comment:
I like the idea of extending File to share the usage of many fields, but I read something in the docs that troubles me from https://developer.nylas.com/docs/intelligence/optical-charecter-recognition/#ocr-response
It says the id field of the response is "IDs of the messages the file is attached to." rather than the File id.  But the response example shows message_ids instead.  If you're able to test, can you check on that?  If the documentation is wrong, can you file an issue to get it fixed?

Reply:
Sorry I forgot to comment on this. The documentation is wrong, it returns the file ID. I'll make sure to make note of this and the other inconsistencies.